Merge branch 'main' of https://github.com/executeautomation/mcp-playwright

Krishnakumar1988 opened the pull request.

executeautomation commented:
Thanks for the PR @Krishnakumar1988 

Can you please help me update the following

1. Tests
2. Documentation for this tool (docusaurus)
3. Screenshot of the test passing in Claude Desktop or Cline or Cursor

Thanks,

Krishnakumar1988 commented:
1. Added the documentation 
 ![image](https://github.com/user-attachments/assets/15ef5b9b-27f6-4077-8450-5cf4adff3425)
2. Added the test 
3. Screenshot of the test passing in VScode

![image](https://github.com/user-attachments/assets/7dbd6b70-0e27-46eb-b660-631ea9c16913)

executeautomation commented:
Merged to main @Krishnakumar1988 , thanks for the feature !
